e2e.py

```diff
@@ -0,0 +1,74 @@
+"""Simple Bigtable client demonstrating auto-generated veneer.
+"""
+
+import argparse
+import logging
+
+from google.bigtable.v1 import bigtable_service_api
+from google.bigtable.admin.cluster.v1 import bigtable_cluster_service_api
+from google.bigtable.admin.table.v1 import bigtable_table_service_api
+
+from google.bigtable.v1 import bigtable_data_pb2 as data
+from google.bigtable.admin.cluster.v1 import bigtable_cluster_data_pb2 as cluster_data
+from google.bigtable.admin.table.v1 import bigtable_table_data_pb2 as table_data
+
+def run(project_id):
+  with bigtable_service_api.BigtableServiceApi() as bigtable_api, \
+       bigtable_cluster_service_api.BigtableClusterServiceApi() as cluster_api, \
+       bigtable_table_service_api.BigtableTableServiceApi() as table_api:
+
+    disp_name = 'my-cluster'
+    zone_name = 'projects/{0}/zones/{1}'.format(project_id, 'us-central1-c')
+    employee_id='employee1'
+
+    try:
+      print 'Creating a cluster.'
+      cluster = cluster_data.Cluster(display_name=disp_name, serve_nodes=3)
+      cluster_name = cluster_api.create_cluster(
+          name=zone_name, cluster_id=disp_name, cluster=cluster).name
+      print 'Successfully created a cluster named {0}'.format(cluster_name)
+
+      print 'Creating a bigtable.'
```

Comment:
This seems fair game to me since the bigtable brand is already in the API. is there a specific change recommended here?